Automattic/wp-calypso
# BackupBrowser: Integrate granular restore flow

This PR is the continuation of the work started on https://github.com/Automattic/wp-calypso/pull/81304.

## Proposed Changes
* Integrate granular restore flow with WPCOM API to restore files using include/exclude paths.
* Fix successful restore page header style

## Screenshots
| Page | Screenshot |
|---|---|
| Confirmation page | <img width="640" alt="CleanShot 2023-09-01 at 16 02 42@2x" src="https://github.com/Automattic/wp-calypso/assets/1488641/04424d90-2646-4d25-8f78-c50a0ec6a358"> |
| Sucessful restore | <img width="646" alt="CleanShot 2023-09-01 at 15 48 46@2x" src="https://github.com/Automattic/wp-calypso/assets/1488641/a4d36b75-f607-42be-a10f-c682f1656894"> |

## Testing Instructions
* Start a Jetpack Cloud or Calypso Live branch and add `?flags=jetpack/backup-granular` to the URL after visiting the Backup Browser.
* Pick a file and ensure the `Restore` button is visible (except on the WordPress item).
* Click on `Restore` and it should redirect you to `/backup/:site/granular-restore/:rewindId`.
* Ensure the file you selected is displayed in the granular restore page.
* Click on `Confirm restore`, it should start the restore process.
  * Validate the active job in the backend to see it is passing properly the include/exclude path.

## Pre-merge Checklist
